### DISCUSSION TITLE
VTreeview Toggle Element - Am I loosing my mind?

### DISCUSSION THREAD

**Discussion by jabbott-the-hut** [vuetifyjs/vuetify#15991] [Q&A] [answered]
Why is it that when I check devtools on the Vuetify doccos page (screenshot below) the node toggle is created using button tags yet in all versions that I have tried so far (up to and including v2.6.12) the same toggle is created using <i> (icon) tags.

For users who must navigate this treeview with just the keyboard, it is not selectable or navigable at all, without quite a bit of fettling.

Am I missing a very obvious part of the puzzle here?

[Vuetify Docco  Page  - Treeview](https://vuetifyjs.com/en/components/treeview/)
![image](https://user-images.githubusercontent.com/83084211/198686243-4b83f364-c116-4b52-9ef9-5be3ae04f87c.png)

[Some random example of v-treeview in use](https://codesandbox.io/s/pjz1w25n8x)
![image](https://user-images.githubusercontent.com/83084211/198687364-52b5ed56-e501-46bb-82de-ab77a7d06904.png)

**Comment by KaelWD** [comment, marked as answer]
"Some random example of v-treeview in use" is using 1.5.1

**Reply by jabbott-the-hut** [reply]
We've figured it out now, but thanks ever so much for your help Kael 🙏
